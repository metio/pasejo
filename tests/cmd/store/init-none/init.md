```console
$ PASEJO_CONFIG=config.toml pasejo store init --path path/to/store --name default --vcs none
info: Store initialized at '[CWD]/path/to/store'

```

```console
$ PASEJO_CONFIG=config.toml pasejo store init --path path/to/another --name another --default --vcs none
info: Store initialized at '[CWD]/path/to/another'
info: Store 'another' is now the default

```

```console
$ PASEJO_CONFIG=config.toml pasejo store init --path existing-store --name existing --vcs none
info: Store initialized at '[CWD]/existing-store'

```
