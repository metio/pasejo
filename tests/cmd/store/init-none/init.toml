bin.name = "pasejo"
args = ["store", "init", "--path", "path/to/store", "--alias", "default"]

[env.add]
PASEJO_CONFIG = "config.toml"
