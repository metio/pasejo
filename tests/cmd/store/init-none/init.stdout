Store initialized at [CWD]/path/to/store
